support/ref-docs/src/index.ts

```diff
@@ -0,0 +1,68 @@
+/// <reference types="./types/remark-toc" />
```

Comment:
🤔 in other packages, we've chosen to keep the `types` directory for locally defined types (as opposed to those installed from `@types/*` packages) at the root of the package instead of inside the `src` directory. is that something you think might be worth changing?

also, we've used the `tsconfig.json` to configure the compiler to look in that directory for packages by name, instead of using `/// <reference .../>` directives. do you feel like this is something we should change?